commands/games/wordle.py

```
@@ -201,90 +253,237 @@ def __init__(self, command_info: utility.CommandInfo, config) -> None:  # type:
+        @discord.ui.button(
+            label=tanjunLocalizer.localize(str(command_info.locale), "commands.games.wordle.buttons.playNormal"),
+            style=discord.ButtonStyle.green,
+        )
+        async def normal_button_callback(
+            self,
+            interaction: discord.Interaction,
+            button: discord.ui.Button[Any],
+        ) -> None:
+            nonlocal hard_mode
+            hard_mode = False
+            await _start_game(interaction)
```

Flake8 (7.3.0)

[error] undefined name '_start_game'

(F821)